Workbooks/ServiceBus/AtResource/ResourceInsights.workbook

```diff
@@ -0,0 +1,2500 @@
+            "description": "Used in 'No Subscriptions' workbook template",
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 0
+            },
+            "timeContextFromParameter": "TimeRange"
+          }
+        ],
+        "style": "above",
+        "queryType": 1,
+        "resourceType": "microsoft.resourcegraph/resources"
+      },
+      "name": "parameters - 0",
+      "styleSettings": {
+        "margin": "15px 0 0 0"
+      }
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "<br/>"
+      },
+      "name": "text - 5"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "template",
+        "loadFromTemplateId": "community-Workbooks/Common/noSubscriptions",
+        "items": []
+      },
+      "conditionalVisibility": {
+        "parameterName": "Subscription",
+        "comparison": "isEqualTo",
+        "value": ""
+      },
+      "name": "No Subscriptions group"
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "tabs",
+        "links": [
+          {
+            "id": "0570ec01-d6bf-495e-a637-8baa17e5c3e3",
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Overview",
+            "subTarget": "Overview",
+            "style": "link"
+          },
+          {
+            "id": "0339c6cd-e2cb-459b-a09f-1c83640bdac9",
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Requests",
+            "subTarget": "Requests",
+            "style": "link"
+          },
+          {
+            "id": "6a954fb7-3294-43a5-8377-b44e6cf6ccba",
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Messages",
+            "subTarget": "Messages",
+            "style": "link"
+          }
+        ]
+      },
+      "name": "links - 2",
+      "styleSettings": {
+        "margin": "-15px 0px -15px 0px"
+      }
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "<br/>"
+      },
+      "name": "text - 4"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 12,
+            "content": {
+              "version": "NotebookGroup/1.0",
+              "groupType": "editable",
+              "items": [
+                {
+                  "type": 10,
+                  "content": {
+                    "chartId": "workbookcd69dced-b884-486e-bba0-793500f1fffe",
+                    "version": "MetricsItem/2.0",
+                    "size": 4,
+                    "chartType": -1,
+                    "resourceType": "microsoft.servicebus/namespaces",
+                    "metricScope": 0,
+                    "resourceParameter": "Resources",
```

Comment:
After renaming to just 'Resource', make sure all these resourceParameter references are updated